lib/pkg.ml

```diff
@@ -518,7 +518,7 @@ let distrib_version_opam_files ~dry_run ~version =
   infer_pkg_names Fpath.(v ".") [] >>= fun names ->
   List.fold_left
     (fun acc name ->
-      acc >>= fun _acc ->
+      acc >>= fun () ->
```

Comment:
Maybe we should extract a `Stdext.Result.List.iter` for this. I'm fine with leaving it as it is but this might be a bit confusing. Hopefully we'll soon be able to use `stdune`!

Reply:
done